com/win32com/client/gencache.py

```diff
@@ -52,7 +56,8 @@
 
 # A dictionary of ITypeLibrary objects for demand generation explicitly handed to us
 # Keyed by usual clsid, lcid, major, minor
-demandGeneratedTypeLibraries = {}
+# Typing as Any because PyITypeLib is not exposed
+demandGeneratedTypeLibraries: dict[tuple[str, int, int, int], Any] = {}
```

Comment:
`Any` should be used sparingly, it's an escape hatch. For instance: When it's impossible to represent a type accurately using the current type system.

In this case, we have no way to represent this type though python code (for now).